open-metadata-test/open-metadata-fvt/access-services-fvt/subject-area-fvt/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/fvt/CategoryFVT.java

```diff
@@ -95,12 +98,12 @@ public static int getCategoryCount(String url, String serverName, String userId)
 
     public void run() throws  SubjectAreaFVTCheckedException, InvalidParameterException, PropertyServerException, UserNotAuthorizedException {
 
-        System.out.println("Create a glossary");
+        log.debug("Create a glossary");
         Glossary glossary = glossaryFVT.createGlossary(serverName + " " + DEFAULT_TEST_GLOSSARY_NAME);
         String glossaryGuid = glossary.getSystemAttributes().getGUID();
-        System.out.println("Create a category1");
+        log.debug("Create a category1");
```

Comment:
can we surround all log.debug with if (log.isDebugEnabled()) so the parameters are no formatted when debug is not on please.

Reply:
Completed